Monika After Story/game/updates.rpy

```diff
@@ -332,6 +332,10 @@ label v0_9_6(version="v0_9_6"):
 
                 # erase this topic
                 mas_eraseTopic(old_ev_label, persistent.event_database)
+
+        # this doesn't need to be locked by default anymore with the new greet code
+        if not seen_event("greeting_tears"):
```

Comment:
```suggestion
        if not renpy.seen_label("greeting_tears"):
```
`seen_event` checks both label and event list, and since this is not during runtime, we don't really need to check event list here.